### PR TITLE
🛠 Update Dependencies

### PR DESCRIPTION
**Pull request by Loopito**
## Updating Dependencies
| Name | Updating | dependencies |
| :----  | :--------: | :-: |
| [@hookform/resolvers](https://github.com/react-hook-form/resolvers/releases) | 1.3.0...1.3.4 | * |
| [immer](https://github.com/immerjs/immer/releases) | 8.0.0...8.0.1 | * |
| [react-hook-form](https://github.com/react-hook-form/react-hook-form/releases) | 6.14.2...6.15.1 | * |
| [react-intl](https://github.com/formatjs/formatjs/releases) | 5.10.16...5.12.0 | * |
| [react-scripts](https://github.com/facebook/create-react-app/releases) | 4.0.1...4.0.2 | * |

Powered by [@elestu/actions-dependacop](https://github.com/elestu/actions-dependacop)

### CODE REVIEW

File: package.json
Comment (nicocadq) [inline package.json:46]: The `@hookform/resolvers` update is breaking our tests and I think our build process too, because they have a problem with their codebase, but they fix it in [this PR](https://github.com/react-hook-form/resolvers/pull/108). Can we wait until this PR is merged and keep it at `1.3.0`?
```suggestion
    "@hookform/resolvers": "1.3.0",
```

**Review by mconiglio — APPROVED** [review]

**Review by ValeFranchi — APPROVED** [review]

**Pull request merged**
